source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
+    - MSI for `32-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x86.msi>`__
 
-3. To create a Desktop launcher, open the file **README.md** and follow the instructions in the **Desktop launcher** section.
+    See the `MSI installer and group policy documentation <https://docs.mattermost.com/install/desktop-msi-gpo.html>`__ for instructions on installing the Mattermost Desktop App via an MSI installer and configuring supported group policies. Changes to group policies require you to restart Mattermost for those changes to take effect.
 
-Ubuntu and Debian-based systems
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    The following group policies are available:
 
-Unofficial, community-driven ``.deb`` packages are available.
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
+    | Group policy               | Description                                                                 | Required Mattermost version     |
+    +============================+=============================================================================+=================================+
+    | Enable Server Management   | If disabled, management of servers in the app settings are disabled.        | 4.3 or later                    |
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
+    | Default Server List        | Define one or more default, permanent servers.                              | 4.3 or later                    |
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
 
-1. Download the latest version of the Mattermost Desktop App:
+Update the Mattermost Desktop App
+---------------------------------
 
-  - 64-bit systems `mattermost-desktop-5.0.4-linux-amd64.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-amd64.deb>`__
-  - 32-bit systems `mattermost-desktop-5.0.4-linux-i386.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-i386.deb>`__
+For the best experience, we recommend keeping your Desktop App updated to the latest release. 
 
-2. At the command line, execute one of the following commands depending on the package that you downloaded:
+.. tabs::
 
-  - 64-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-amd64.deb``
-  - 32-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-i386.deb``
+  .. tab:: Linux
 
-3. Run Mattermost as a desktop app.
+    Depending on the distribution you're using, your Desktop App updates automatically when a new release is available.
 
-Linux rpm files (Beta)
-~~~~~~~~~~~~~~~~~~~~~~
+    - Ubuntu: A Debian repository is available to configure and install. TBD
```

Comment:
@devinbinnie - Can you provide these details when available, please?

Reply:
See above (sorry didn't see this comment yet lol)